devel/ci/integration/ipsilon/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM fedora
```

Comment:
we should probably tag this so that we are not surprised when the python version changes. I think `FROM fedora:33` would be better.